Bottom: ee42d52e3785e628fb49c288ad8fdd02207cd0ec
Top:    b00e7c90c4e7fa6aac628c394c56aebd28ceab67
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-23 00:13:20 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
